### DISCUSSION TITLE
Volkszähler Hichi Smartmeter

### DISCUSSION THREAD

**Discussion by gizmo2006** [hoylabs/OpenDTU-OnBattery#273] [Q&A] [answered]
Ich lese mein Stromverbrauch mit Volkszähler Hichi Smartmeter aus,
das ganze über Tasmota und über MQTT zu ioBroker,

wenn ich in Tasmota die IP  http://192.168.3.xxx/cm?cmnd=status%208 eingebe bekomme ich diese Meldung unter Rohdaten:

{"StatusSNS":{"Time":"2023-06-24T12:53:02","SML":{"Verbrauch_Summe":3503.124,"Einspeisung_Summe":434.212,"Watt_L1":-292.55,"Watt_L2":-507.89,"Watt_L3":112.76,"Watt_Summe":-687.69,"Volt_L1":236.8,"Volt_L2":237.6,"Volt_L3":237.7}}}

Meine Frage: Was trage ich in den Http Pfad und in den json ein?   
Http  Pfad=  ? (http://192.168.3.xxx/cm?cmnd=status%208 vom Tasmota?)
Json Pfad = ?

Bei Shelly kommt ja: Json Pfad = total_power
Aber bei Volkszähler Hichi Smartmeter?

Danke

**Comment by spcqike** [comment]
Der http Pfad sollte stimmen. Json sollte 

    StatusSNS/SML/Watt_Summe

sein

**Reply by gizmo2006** [reply]
Danke erstmal, werde es Morgen Testen, jetzt Familie vorrangig,

**Reply by gizmo2006** [reply]
Super, es zeigt Success! Power: -455.83W

**Reply by madmartin** [reply, marked as answer]
Bitte die Frage hier als "beantwortet" markieren...
